app/components/create-vhost/template.hbs

```diff
@@ -20,59 +20,122 @@
             </div>
 
-          {{#unless serviceHasDefaultVhost}}
               <div class="form-group">
-                <label class="block" for="domain-type">Endpoint type:</label>
+                <label class="block" for="domain-type">
+                  Endpoint type:
+                    {{more-info-icon
+                    title="Choosing an Endpoint Type"
+                    bs-html=true
+                    body="
+                    <p>
+                    For development apps and non-user facing APIs, choose a
+                    <strong>Default Endpoint</strong>: Aptible takes care of
+                    the domain name and SSL certificate for you.
+                    </p>
+                    <p>
+                    For other use cases, a <strong>custom domain with managed
+                    SSL</strong> is a good option: you use your own domain
+                    name, but Aptible handles provisioning and renewing your SSL
+                    certificate.
+                    </p>
+                    <p>
+                    For more advanced use cases, like using an Extended
+                    Validation SSL certificate, use a <strong>custom domain
+                    with a custom SSL certificate</strong> for complete control.
+                    </p>"}}
+
+                </label>
                 <div class="radio">
                   <div>
-                    {{#radio-button value=true groupValue=model.isDefault name="domain-type"}}
-                      {{#bs-tooltip placement="bottom" title='A default endpoint has an endpoint address in the format app-${id}.on-aptible.com.' bs-container=false}}
-                          Use app-{{ model.app.id }}.on-aptible.com default endpoint address
-                      {{/bs-tooltip}}
-                    {{/radio-button}}
+                    {{#if defaultVhostAllowed}}
+                      {{#radio-button value=TYPE_DEFAULT groupValue=vhostType name="domain-type"}}
+                        Use app-{{ model.app.id }}.on-aptible.com default endpoint.
+                      {{/radio-button}}
+                    {{else}}
+                      <label class="disabled">
+                        <input type="radio" class="ember-view" disabled>
+                        Default endpoint unavailable: this app already has one.
+                      </label>
+                    {{/if}}
+                  </div>
+
+                  <div>
+                    {{#if acmeVhostAllowed}}
+                      {{#radio-button value=TYPE_ACME groupValue=vhostType name="domain-type"}}
+                        Use a custom domain with managed SSL.
+                      {{/radio-button}}
+                    {{else}}
+                      <label class="disabled">
+                        <input type="radio" class="ember-view" disabled>
+                        Managed SSL unavailable: this feature is not available
```

Comment:
Sure; I think I'm using "legacy stacks" in metrics. I can change that. Do we refer to v1 explicitly anywhere else? (I don't recall if we have a tooltip explaining why scaling isn't available on v1)